src/pages/projectCreate/components/ProjectCreateLaunchedModal.tsx

```diff
@@ -89,8 +90,8 @@ export const ProjectCreateLaunchedModal = (
               onClick={props.onClose}
               leftIcon={<NostrSvgIcon boxSize={'20px'} />}
               textDecoration={'none'}
-              color="neutral.1000"
-              bgColor="social.nostr"
+              color={lightModeColors.neutral[0]}
```

Comment:
If you are specifically calling lightmode colors, does this render well with contrast in darkmode?  Just double check and make sure - I've done this and then darkmode looked terrible for me.

Reply:
That was actually the requirement for the task, but I think I see another mistake here, I should also make the bgColor static for this one to avoid confusion.